### ISSUE TITLE
groundSSD i_other may be undefined

### ISSUE THREAD

**Issue by jmcfarland-swri** [jmcfarland-swri/NASA_ULI_InfoFusion#7]
@edecarlo-swri The first branch in this `if` statement does not define `i_other`, but `i_other` is used immediately after the statement on line 351.

https://github.com/jmcfarland-swri/NASA_ULI_InfoFusion/blob/daacf38b9369723aec0ee007040b8886a58749bd/src/PARA_ATM/safety/ground_ssd.py#L331-L351

It seems that if the code goes into the first branch, it is either going to result in an error or end up using a value of `i_other` from a previous iteration (which I'm guessing is not the intent).

**Comment by edecarlo-swri** [comment]
You are right, that is not the intent. It looks like a lot of computation could be skipped (all the way to Line 456?) in the case of len(ind)=0.
